providers/defltprov.c

```diff
@@ -534,6 +534,14 @@ static const OSSL_ALGORITHM deflt_keymgmt[] = {
     { NULL, NULL, NULL }
 };
 
+static const OSSL_ALGORITHM deflt_skeymgmt[] = {
+    { PROV_NAMES_AES, "provider=default", ossl_aes_skeymgmt_functions,
+      PROV_DESCS_AES },
+    { PROV_NAMES_GENERIC, "provider=default", ossl_generic_skeymgmt_functions,
```

Comment:
Since we've got a generic implementation, should the default for fetching SKEYMGMT objects when their name hasn't been specified be:
1. algorithm name
2. generic implementation